internal/remote/tcp_conn.go

```diff
@@ -40,6 +48,19 @@ func initConn(ctx context.Context, addr string) (*tcpConnWrapper, error) {
 	}, nil
 }
 
+// reference by net.ParseIP
+func isIPV4(s string) bool {
+	for i := 0; i < len(s); i++ {
+		switch s[i] {
+		case '.':
+			return true
```

Comment:
for a domain like www.nameserver.com:8080, shall we consider it as an IPv4 string?

Reply:
we just justify ipv6 or  ipv6, domain name never runn into here

Reply:
OK, Got it. 
The native branch has been moved to the master, you can submit this PR directly to the master。